3-networks-dual-svpc/modules/base_env/outputs.tf

```diff
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
+output "target_name_server_addresses" {
+  value       = var.target_name_server_addresses
+  description = "List of IPv4 address of target name servers for the forwarding zone configuration"
```

Comment:
NIT: add a more details about this variable description here, would this be an on-prem router or on-prem name server, or other?

Reply:
Forwarding_zones are related to name server. I will add more details here.